### DISCUSSION TITLE
Lag spikes after update

### DISCUSSION THREAD

**Discussion by DatVeno** [ppy/osu#20612] [Q&A] [answered]
### Type

Performance

### Bug description

So I wanted to try lazer after a while again, but I have lagspikes since I updated to the current version. I already tried to disable Shadowplay, Discord overlay is off, tried different skins, frame limiter and the task manager doesnt show any spikes that could cause this.

### Screenshots or videos

https://user-images.githubusercontent.com/38132190/194703798-b6b84c62-69d1-4ca7-a426-d48db6f6b5e9.mp4

### Version

2022.1008.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9739016/database.log)
[network.log](https://github.com/ppy/osu/files/9739017/network.log)
[performance.log](https://github.com/ppy/osu/files/9739018/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9739019/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9739021/updater.log)

**Comment by Loreos7** [comment]
Highly agree.

https://user-images.githubusercontent.com/86934170/194706107-9c968f03-6ac6-474a-97e3-0eec5561b5c7.mp4


https://user-images.githubusercontent.com/86934170/194706109-2e2c7931-1761-452f-8a95-cdd05015ff04.mp4

**Reply by Loreos7** [reply]
FIXED by removing this from the skin. Again.
![image](https://user-images.githubusercontent.com/86934170/194708556-7c71ce9f-3068-4502-9e32-4bcd253f3bdb.png)

**Comment by peppy** [comment, marked as answer]
Will be fixed in a hotfix release.
